src/Sulu/Bundle/CategoryBundle/Controller/CategoryController.php

```diff
@@ -342,7 +350,7 @@ protected function getListRepresentation(
         );
     }
 
-    private function initializeListBuilder($locale, bool $defaultSort = false)
+    private function initializeListBuilder($locale, bool $defaultSort = false): ListBuilderInterface
```

Comment:
Not a bc break since this is private, even if the factory would return a non doctrine list builder.